.github/workflows/pinned-gh-actions.yml

```diff
@@ -0,0 +1,24 @@
+name: Check SHA-pinned GitHub Actions
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
```

Comment:
drop, you don't need `edited`, that's for rerunning it when the comment is edited, the other three are the default, I'm taking that you copied this block from the compliance workflow, there's a step there that checks that the comment is not empty so it needs edited to work correctly but nothing else should have it

Reply:
> I'm taking that you copied this block from the compliance workflow

bingo :)

Reply:
I meant drop the whole block, the one you need is the default, just

```
on:
  pull_request:
    paths:
...
```

Reply:
ah - done